### PR TITLE
Update Anonymous Proxy with Note about type `Any`

### LINKED ISSUE #2594: Update Anonymous Proxies article
There should be a warning in the anonymous proxy section that tells users the first proxy they add should always be of type "Any". Also, if there are multiple proxies for the proxied account, they shouldn't remove all `Any` types unless they're ready to kill the relationship.

Without an `Any` type they won't be able to send funds, add new proxies or kill the anonymous proxy or take any action not specifically allowed by the types of the proxies the account has.

### PR DESCRIPTION
**Pull request by salmad3**

### CODE REVIEW

**Comment by DrW3RK** [comment]
@michalisFr let us know if this addresses the issue

File: docs/learn/learn-proxies.md [hunk 1]
```diff
@@ -105,6 +105,13 @@ multisig would be created.
 
 ## How to set up a Proxy
 
+> NOTE: The first proxy you add should always be of _type_ `Any`.
```
Comment (michalisFr) [inline docs/learn/learn-proxies.md]: This warning applies only to Anonymous proxies, so it should go under `Creating Anonymous Proxies`.

Comment (salmad3) [reply docs/learn/learn-proxies.md]: Updated

**Comment by salmad3** [comment]
Merging after addressing feedback.

**Pull request merged**